### PR TITLE
Des 577

### PR DESCRIPTION
**Pull request by StommePoes**
Combining DES-577 (radio buttons) and since I had to update inputs version anyway, DES-582 (update main Elements page to match README), and also updated the README.

Additionally while my terminal was open, Tory notices a duplication of button padding for the large buttons. The duplicate was also incorrect for large buttons (12px padding instead of 20px).

### CODE REVIEW

File: docs/elements/index.md [hunk 1]
```diff
@@ -8,25 +8,37 @@ title: Getting started with Elements
...
- correct location in your web server, and ensure your application is set up to allow the font files to download correctly 
- and not filter out the response header Content Type (i.e. 'application/x-font-ttf') so the icons may be rendered in the
-  browser.
+Some caching headers will prevent Internet Explorer from showing font files after a refresh. The culprits are `Cache-control: no-store` and on https `Pragma: no-cache`.
+
+#### Icons
+
+This version of Elements uses an SVG sprite for icons (the Pearson logos are two separate SVGs, not part of the sprite). The sprite file is found in `/dist/icons/` in Elements, and should be copied to the appropriate place, as mentioned above, in your web server. In additional, to support Internet Explorer and some versions of Android where the default-Chrome browser is actually webkit, that SVG sprite must be present on your application's page. See <a href="https://pearson-elements-v1.surge.sh/elements/icons/">the icons page</a> under "Two methods of including icons" for examples of getting this sprite on your application page. Additionally, there are instructions on that page for how to add new icons.
```
Comment (JasonLantz) [inline docs/elements/index.md]: surge or gh_pages?

File: docs/elements/index.md [hunk 2]
```diff
@@ -42,8 +54,7 @@ Clone the project and install the dependencies:
 
 ### Adding Icons
 
-Please follow the steps outlined in the [how-to-add-an-icon](how-to-add-an-icon.md) file to add an icon. This assumes the icon is a Font-Awesome icon and was already approved by UX Design (contact the [Elemental Design Google Group](https://groups.google.com/a/pearson.com/forum/#!forum/elemental-discuss/categories)).
-
+See <a href="https://pearson-elements-v1.surge.sh/elements/icons/">the icons documentation</a> under "Adding new icons" for the process of adding a new icon.
```
Comment (JasonLantz) [inline docs/elements/index.md]: same...

Comment (JasonLantz) [inline docs/elements/index.md]: surge or gh?

Comment (StommePoes) [inline docs/elements/index.md]: if github, you need to tell me the URL because I can't, and can never, find it. I still use surge.

**Comment by StommePoes** [comment]
\o/

**Comment by udhadpa** [comment]
Tests written and verified. Good to merge

**Pull request merged**